docs/current_docs/api/chaining.mdx

```diff
@@ -41,49 +41,49 @@ dagger -m github.com/shykes/daggerverse/wolfi@v0.1.4 call container with-exec --
 Here is an example of chaining a `Container.terminal()` function call to start an interactive terminal in the container returned by a Wolfi container builder Dagger Function:
 
 ```shell
-dagger -m github.com/shykes/daggerverse/wolfi@v0.1.4 call container --packages=cowsay terminal
+dagger -m github.com/dagger/dagger/modules/wolfi@v0.16.2 -c 'container --packages=cowsay | terminal'
 ```
 
 By default, the terminal is started with the `sh` shell, although this can be overridden by adding the `--cmd` argument. To start the same terminal with the `zsh` shell, use:
 
 ```shell
-dagger -m github.com/shykes/daggerverse/wolfi@v0.1.4 call container --packages=cowsay,zsh terminal --cmd=zsh
+dagger -m github.com/dagger/dagger/modules/wolfi@v0.16.2 -c 'container --packages=cowsay,zsh | terminal --cmd=zsh'
 ```
 
 ## Export directories, files and containers
 
 Just-in-time artifacts such as containers, directories and files can be exported to the host filesystem from the Dagger Function that produced them. The destination path on the host is specified using the `--path` argument.
 
-Here is an example of exporting the directory returned by a Go builder Dagger Function to the `./my-build` directory on the host:
+Here is an example of exporting the build directory returned by a Go builder Dagger Function to the `./my-build` directory on the host:
 
 ```shell
-dagger -m github.com/kpenfound/dagger-modules/golang@v0.2.0 call build --source=https://github.com/dagger/dagger --args=./cmd/dagger export --path=./my-build
+dagger -m github.com/kpenfound/dagger-modules/golang@v0.2.1 -c 'build ./cmd/dagger --source=https://github.com/dagger/dagger  | export ./my-build'
```

Comment:
`--source` is optional in the referenced module but the command fails if you omit it, so maybe it should be required? Ping @kpenfound for review

Reply:
@kpenfound can you please help with this? Maybe update the source module?